context/app/static/js/components/Routes.jsx

```diff
@@ -3,15 +3,13 @@ import PropTypes from 'prop-types';
 import Container from '@material-ui/core/Container';
 import { Home } from './Home';
 import Search from './Search/Search';
-import Donor from './Detail/Donor';
-import Sample from './Detail/Sample';
-import Dataset from './Detail/Dataset';
+import { Donor, Sample, Dataset, Collection } from './Detail';
 import Showcase from './Showcase';
-import Collections from './Collections';
+import { Collections } from './Collections';
```

Comment:
I expect capitalized imports to have default export... if you have some other pattern in mind, that can work, but let's talk.

Reply:
I have been trying to consistently use named exports in domain level `index.js` files throughout. There will be situations where we have multiple exports like `Detail` pages and I thought it best to be consistent.

Reply:
if there's not a default, could the filename be lowercase?